src/shared/common/forms/composite/somu-list.jsx

```diff
@@ -105,6 +105,31 @@ class SomuTableRenderer {
         </>);
     }
 
+    renderFoiAppeal(somuItem) {
+        const { choices } = this.state;
+        const title = `${this.loadValue(somuItem.appealType, choices)}`;
+
+        return (<>
+            <td className='govuk-table__cell'>
+                <label className={'govuk-label'}
+                    aria-label={title}
+                    title={title}>
+                    {title}
+                </label>
+            </td>
+            <td className='govuk-table__cell'>
+                <label className={'govuk-label'}
+                    aria-label={somuItem.appealStatus === 'appealComplete' ? 'Complete': 'Pending'}
+                    title={somuItem.appealStatus === 'appealComplete' ? 'Complete': 'Pending'}>
+                    {somuItem.appealStatus === 'appealComplete' ? 'Complete': 'Pending'}
+                </label>
+            </td>
+        </>
+        );
+    }
+
+
+    renderStatusColumn(status, { contributionDueDate }) {
```

Comment:
Whoops... what's happened here? Merge/Rebase error?

Reply:
I have addressed this